### PR TITLE
Remove set -x from tests/setup-runtime-dependencies.sh.

### PR DESCRIPTION
**Pull request by adityamandaleeka**
I hadn't run runtest.sh in a while. It looks like it's now calling this script which is adding a bunch of noise to the output.

@sejongoh Please let me know if there is any reason you left the `set -x` in.

### CODE REVIEW

**Comment by adityamandaleeka** [comment]
@sejongoh PTAL

File: tests/setup-runtime-dependencies.sh [hunk 1]
```diff
@@ -96,7 +97,7 @@ if [ ! -e $dotnetToolsDir ]; then
     exit_with_error 1 'Directory containing dotnet commandline does not exist:'$dotnetToolsDir 
 fi
 if [ ! -e $dotnetCmd ]; then
-    exit_with_error 1 'donet commandline does not exist:'$dotnetCmd
+    exit_with_error 1 'dotnet commandline does not exist:'$dotnetCmd
```
Comment (jasonwilliams200OK) [inline tests/setup-runtime-dependencies.sh]: Both of these typos also exist in `setup-runtime-dependencies.cmd`.

**Comment by sejongoh** [comment]
LGTM

**Comment by adityamandaleeka** [comment]
Thanks @jasonwilliams200OK. I've fixed them there as well.

**Comment by adityamandaleeka** [comment]
@dotnet-bot test this please

**Pull request merged**